dvc/remote/gdrive.py

```diff
@@ -155,51 +160,89 @@ def _validate_config(self):
             or not self._service_account_p12_file_path
         ):
             raise DvcException(
-                "To use service account please specify {}, {} and "
-                "{} in DVC config. Learn more at "
-                "{}.".format(
-                    "gdrive_service_account_email",
-                    "gdrive_service_account_p12_file_path",
-                    "gdrive_service_account_user_email (optional)",
-                    format_link("https://man.dvc.org/remote/modify"),
+                "To use service account set 'gdrive_service_account_email',\n"
+                "'gdrive_service_account_p12_file_path', and optionally "
+                "'gdrive_service_account_user_email'\nin DVC config. "
+                "Learn more at {}".format(
+                    format_link("https://man.dvc.org/remote/modify")
                 )
             )
 
         # Validate OAuth 2.0 Client ID configuration
         if not self._use_service_account:
             if bool(self._client_id) != bool(self._client_secret):
                 raise DvcException(
-                    "Please specify Google Drive's client id and secret in "
-                    "DVC config or omit both to use defaults. Learn more at "
-                    "{}.".format(
+                    "Please specify GDrive's client id and secret in "
+                    "DVC config or omit both to use defaults.\n"
```

Comment:
```suggestion
                    "Please specify GDrive's client ID and secret in "
                    "DVC config or omit both to use the defaults.\n"
```

Reply:
We use upper case "ID" in the GDrive guide, https://dvc.org/doc/user-guide/setup-google-drive-remote:

![image](https://user-images.githubusercontent.com/1477535/80438554-111b6200-88ca-11ea-8333-2ecd9377e3d8.png)

![image](https://user-images.githubusercontent.com/1477535/80438564-18427000-88ca-11ea-8e42-ab435e98be50.png)